anomalous-token-transfers-ml-py/src/utils/constants.py

```diff
@@ -0,0 +1,58 @@
+ANOMALY_THRESHOLD = 0.5
+
+ERC20_TRANSFER_EVENT = '{"name":"Transfer","type":"event","anonymous":false,"inputs":[{"indexed":true,"name":"from","type":"address"},{"indexed":true,"name":"to","type":"address"},{"indexed":false,"name":"value","type":"uint256"}]}'
+
+ETHPLORER_KEY = "BACON"
+ETHPLORER_ENDPOINT = "https://api.ethplorer.io"
+
+LUABASE_ENDPOINT = "https://api.luabase.com/run"
+LUABASE_QUERY_UUID = "BACON"
+
+MODEL_CREATED_TIMESTAMP = "1657669403" # Tuesday, July 12, 2022 11:43:23 PM
+MODEL_FEATURES = [
+    'APE_transfers',
+    'APE_value',
+    'CRV_transfers',
+    'CRV_value',
+    'DAI_transfers',
+    'DAI_value',
+    'GALA_transfers',
+    'GALA_value',
+    'HEX_transfers',
+    'HEX_value',
+    'KOK_transfers',
+    'KOK_value',
+    'LINK_transfers',
+    'LINK_value',
+    'LOOKS_transfers',
+    'LOOKS_value',
+    'MANA_transfers',
+    'MANA_value',
+    'MATIC_transfers',
+    'MATIC_value',
+    'SAITAMA_transfers',
+    'SAITAMA_value',
+    'SAND_transfers',
+    'SAND_value',
+    'SHIB_transfers',
+    'SHIB_value',
+    'SOS_transfers',
+    'SOS_value',
+    'STRNGR_transfers',
+    'STRNGR_value',
+    'STRONG_transfers',
+    'STRONG_value',
+    'USDC_transfers',
+    'USDC_value',
+    'USDT_transfers',
+    'USDT_value',
+    'WBTC_transfers',
+    'WBTC_value',
+    'WETH_transfers',
+    'WETH_value',
+    'account_age_in_minutes',
+    'max_single_token_transfers',
```

Comment:
can you double check this list; seems some inconsistencies with the data processing function that creates the features

Reply:
yup, matching the feature name will fix it.